core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -3078,6 +3089,17 @@ public static double asin(double b0) {
     return Math.asin(b0);
   }
 
+  // ASIND
```

Comment:
Frankly, the little comment is not particularly useful

Reply:
This is following a pattern in the `SqlFunctions` source file. It is meant to start the group of `ASIND` methods. Lower down there will be a `// ASINH` comment that starts the list of `ASINH` methods.